config/default.yaml

```diff
@@ -325,6 +325,7 @@ instance:
   # By default, "do_not_list" or "blur" or "display" NSFW videos
   # Could be overridden per user with a setting
   default_nsfw_policy: 'do_not_list'
+  federate_unlisted_videos: true
```

Comment:
Please put this in a dedicated key, above https://github.com/Chocobozzz/PeerTube/blob/develop/config/production.yaml.example#L183:

```
federation:
  videos:
    federated_unlisted: true
```

Reply:
If you want admins to be able to update this setting using the web interface, then put this at the end of the produdction.yaml file. Don't forget to also update the default.yaml file